Doc/library/math.rst

```diff
@@ -175,6 +175,27 @@ Number-theoretic and representation functions
    of *x* and are floats.
 
 
+.. function:: remainder(x, y)
+
+   Return the IEEE 754-style remainder of ``x`` with respect to ``y``. For
```

Comment:
There is a bit of inconsistency in the file, but I see that in general the parameter do not have a code markup bat italic: *x* and *y* instead of `x` and `y`. 
I also see in general there are two whitespaces after the period.